mlir-sycl/test/Conversion/SYCLToLLVM/sycl-methods-to-llvm.mlir

```diff
@@ -1061,31 +1061,89 @@ func.func @test(%group: memref<?x!sycl_group_1_>, %i: i32) -> i64 {
 !sycl_range_3_ = !sycl.range<[3], (!sycl.array<[3], (memref<3xi64, 4>)>)>
 !sycl_group_3_ = !sycl.group<[3], (!sycl_range_3_, !sycl_range_3_, !sycl_range_3_, !sycl_id_3_)>
 
+module attributes {gpu.container} {
+  gpu.module @kernels {
```

Comment:
We need to wrap this as we see to be able to lower to `spirv` in the intermediate step.